### PR TITLE
New interface design

### PR DESCRIPTION
**Pull request by raffij**
Update interface to better suit the new text block functionality.

### CODE REVIEW

**Comment by raffij** [comment]
Fixing firefox test

File: package.json [hunk 1]
```diff
@@ -13,7 +13,8 @@
   },
   "dependencies": {
     "es5-shim": "^4.0.3",
-    "selection-range": "^1.0.1"
+    "selection-range": "^1.0.1",
+    "url-loader": "^0.5.7"
```
Comment (callum) [inline package.json]: This should be a development dependency.

Comment (raffij) [inline package.json]: Yes they should.

**Comment by callum** [comment]
Looks good to me

**Pull request merged**